index.html

```diff
@@ -1,59 +1,76 @@
-            Most Notably, I hope to include the following:
-        </p>
-
-        <ul class="centerAlign">
-            <li>Fully Functional Shopping Cart.</li>
-            <li>Payment Method.</li>
-            <li>Shipping, Taxes, and Total Price Calculated.</li>
-            <li>Email Subscription Section.</li>
-            <li>Modern, streamlined style.</li>
-        </ul>
-
-        <p class="centerAlign">
-            After talking with other people within ravine, I finalized
-            a clothing website due to the fact that there are a lot of 
-            different things that can be learned and encorporated into the site.<br>
-            An e-commerce site is also very common within industries and share many elements that can be 
-            used in other sites as well.
-            It fully utilizes HTML, CSS, and Javascript which will<br>
-            give me the ability to get me more up close and personal with
-            web development as a whole.
-        </p>
-
-        <img src="cart.jpeg" style="width: 350px;height: 300px;">
-        <img src="mail.png" style="width: 600px;height: 300px;">
-        <img src="Math.jpeg" style="width: 500px;height: 300px;">
-        <img src="Payments.jpeg" style="width: 900px;height: 300px;">
-    
-    </body>
+<!--Bootstrap link-->
+<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+<link rel="stylesheet" href="/Users/khanmohammed1/Documents/ravine-projects/mnkhan98.github.io/index.css" type="text/css">
+
+<style>
+  .row div{
+    border: 0.5px solid black; 
+    background: rgb(2, 2, 2);
+    padding: 20px;
+    text-align: center;
+    color: snow;
+    font-family: Verdana, Geneva, Tahoma, sans-serif;
+    }
+
+    body{
+      background: rgb(3, 3, 3);
+    }
+</style>
+
+
+
+<div class="container-fluid">
+  <div class='row'>     
+    <div class ='col-md-1'> 
+      Home    
+    </div>  
+  
+    <div class ='col-md-2'>        
+      Book appointment
+    </div>  
+
+    <div class ='col-md-2'>        
+      shop 
+    </div>  
+
+    <div class ='col-md-5'> 
+      Search  
+    </div>  
+  
+    <div class ='col-md-2'> 
+      cart    
+    </div>  
+  </div>
 </div>
 
-</html>
 
 
 
+
+
+<!--<nav class="navbar navbar-expand-md navbar-light">
+    <div class="container-fluid">
+      <a class="navbar-brand" href="#">Kufi Clothing</a>
+    <nav class="row">
+      < !--split each tag into a column-- >
+    </nav>
+      <div class="collapse navbar-collapse" id="navbarNavAltMarkup">
+        <div class="navbar-nav">
+          <a class="nav-link active" aria-current="page" href="#">Home</a>
+          <a class="nav-link" href="#">Features</a>
+          <a class="nav-link" href="#">Pricing</a>
+        </div>
+      </div>
+    </div>
+  </nav>
+-->
+<html>
+<body>
+
+<h1>.</h1>
```

Comment:
what are these lines for the periods?

Reply:
just left them there because i thought id eventually add more things to it

Reply:
kk